ql/src/java/org/apache/hadoop/hive/ql/exec/repl/bootstrap/load/table/LoadPartitions.java

```diff
@@ -159,13 +159,6 @@ private ReplicationState initialReplicationState() throws SemanticException {
   }
 
   private TaskTracker forNewTable() throws Exception {
-    Database parentDb = context.hiveDb.getDatabase(tableDesc.getDatabaseName());
-    // If table doesn't exist, allow creating a new one only if the database state is older than the update.
-    // This in-turn applicable for partitions creation as well.
-    if ((parentDb != null) && (!event.replicationSpec().allowReplacementInto(parentDb.getParameters()))) {
-      return tracker;
-    }
```

Comment:
Due to this check, external tables are not bootstrapped as DB is later (higher last_repl_id) than table itself. Also, this check doesn't make sense as it is bootstrap flow where we set DB's last_repl_id as 0 before triggering tables bootstrap. It will be always true. Also, in retry case, we depend on checkpoints to ignore replicated tables not this.